lib/srv/desktop/rdp/rdpclient/src/rdpdr/tests.rs

```diff
@@ -47,9 +48,134 @@ fn test_to_windows_time() {
 
 fn client(with_scard_id: bool, established_contexts: u32) -> Client {
     let mut c = Client::new(Config {
-        cert_der: vec![],
-        key_der: vec![],
-        pin: "".to_string(),
+        cert_der: vec![
```

Comment:
These are the real values from my local setup, which I used in order to avoid any problems with decoding. Since it's never opened to an external network I figure there's no harm in checking the key in.